tests/src/test/java/org/jboss/windup/tests/application/WindupArchitectureMediumBinaryModeTest.java

```diff
@@ -185,8 +185,11 @@ private void validateReportIndex(GraphContext context)
         TestReportIndexReportUtil util = new TestReportIndexReportUtil();
         util.loadPage(appReportPath);
 
-        Assert.assertTrue(util.checkIncidentByCategoryRow("optional", 301, 2306));
+        Assert.assertTrue(util.checkIncidentByCategoryRow("optional", 291, 2306));
+        Assert.assertTrue(util.checkIncidentByCategoryRow("mandatory", 0, 0));
+        Assert.assertTrue(util.checkIncidentByCategoryRow("information", 10, 0));
         Assert.assertTrue(util.checkIncidentByCategoryRow("cloud-mandatory", 3, 3));
+        Assert.assertTrue(util.checkIncidentByCategoryRow("mandatory", 0, 0));
```

Comment:
this seems a duplicate so it can be removed or (maybe better) changed to `potential`.